pkg/build/builder/dockerutil.go

```diff
@@ -62,15 +81,27 @@ func pushImage(client DockerClient, name string, authConfig docker.AuthConfigura
 	if glog.V(5) {
 		opts.OutputStream = os.Stderr
 	}
-	var err error
+
 	var retriableError = false
 
+	eventWatch, err := watchForFailedPushEvent(eventClient, ref.Namespace, ref.Name)
```

Comment:
We don't have to fix that issue to merge this PR.  What's the size of a
patch that we would have to carry to properly surface that error?

On Thu, Apr 7, 2016 at 11:57 AM, Michal Minar <notifications@github.com>
wrote:

> In pkg/build/builder/dockerutil.go
> <https://github.com/openshift/origin/pull/8195#discussion_r58897036>:
>
> >  	var retriableError = false
> >
> > +	eventWatch, err := watchForFailedPushEvent(eventClient, ref.Namespace, ref.Name)
>
> Ok, I'll try to negotiate once again.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8195/files/182833487b48f39d14835bb857f60d2f9aad8099#r58897036>
>